plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/util/ConstantPoolHelpers.kt

```diff
@@ -44,17 +44,25 @@ internal fun ClassWriter.findClassReader(): ClassReader? {
     return (classReaderField.get(symbolTable) as? ClassReader)
 }
 
+internal fun ClassReader.getSimpleClassName(): String {
+    return className.substringAfterLast("/")
+}
+
 /**
  * Looks at the constant pool entries and searches for R8 markers
  */
 internal fun ClassReader.isMinifiedClass(): Boolean {
-    val charBuffer = CharArray(maxStringLength)
+    return isR8Minified(this) || classNameLooksMinified(this.getSimpleClassName(), this.className)
```

Comment:
Since there are many different ways of checking it now, I am wondering if we'd not have false positives.

Reply:
Do we call our logger in case `isMinifiedClass` returns true? Let's do it if not, so people know why something isn't being instrumented.

Reply:
Log statement is in place:
```
SentryPlugin.logger.info {
                "$simpleClassName skipped from instrumentation because it's a minified class."
            }
```

I guess it's better to skip instrumentation for too many classes rather than break the build. But yes, we'll surely have some false positives due to the regex.